src/applications/ivc-champva/10-7959f-2/components/UploadDocuments.jsx

```diff
@@ -16,7 +16,7 @@ export const supportingDocsInfo = (
       <li>COVID-19 vaccines</li>
     </ul>
     <a href="https://www.va.gov/resources/how-to-file-a-va-foreign-medical-program-claim/#supporting-documents-to-send-w">
```

Comment:
This one will need to be updated to open in a new tab

```suggestion
    <a href="https://www.va.gov/resources/how-to-file-a-va-foreign-medical-program-claim/#supporting-documents-to-send-w" rel="noreferrer" target="_blank">
```